aws_route53_record.cname.tf

```diff
@@ -0,0 +1,15 @@
+/*resource "aws_route53_record" "cname_1" {
+  zone_id = var.zone_id
+  name    = "amq-broker-1.tf.8x8.dev"
```

Comment:
maybe parameterize the top level domain being used that matches the Zone ID. `tf.8x8.dev` as an attribute